app/uk/gov/hmrc/securemessage/models/core/MessageFilter.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.securemessage.models.core
+
+import play.api.libs.json.{ Json, Reads }
+import uk.gov.hmrc.common.message.model.Regime
+
+case class MessageFilter(
+  taxIdentifiers: Seq[String] = List(),
+  regimes: Seq[Regime.Value] = List(),
+  countOnly: Boolean = false)
```

Comment:
We should remove this count, as there is a separate Endpoint for this

Reply:
Good catch, overlooked by me, thanks